app/assets/javascripts/select-kit/components/tag-drop.js.es6

```diff
@@ -119,6 +124,21 @@ export default ComboBoxComponent.extend({
     }
   },
 
+  _prepareSearch(query) {
+    const data = {
+      q: query,
+      limit: this.get("siteSettings.max_tag_search_results")
+    };
+
+    this.searchTags("/tags/filter/search", data, this._transformJson);
+  },
+
+  _transformJson(context, json) {
+    let results = json.results;
+    results = results.sort((a, b) => a.id > b.id);
```

Comment:
I'm curious why we need to sort this and if possible, can we sort this server side instead?